code/headers/frame_handler.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <array>
+#include <display_ostream.hpp>
+
+#include <battery_frame_action.hpp>
+#include <distance_frame_action.hpp>
+#include <manual_control_frame_action.hpp>
+#include <temperature_frame_action.hpp>
+
+namespace r2d2::robos {
+    class frame_handler_c {
+    private:
+        base_comm_c &comm;
+
+        display_ostream_c power_ostream{
+            comm, claimed_display_cursor::ROBOS_POWER_CURSOR, 0, 0};
+        display_ostream_c distance_ostream{
+            comm, claimed_display_cursor::ROBOS_DISTANCE_CURSOR, 0, 16};
+        display_ostream_c temperature_ostream{
+            comm, claimed_display_cursor::ROBOS_TEMPERATURE_CURSOR, 0, 32};
+
```

Comment:
Why are all the different types hardcoded? I'd imagine there's another way to handle this without having to change RobOS each time there's a new frame

Reply:
According to the previous team this could be changed, by using static members. Our team is planning to change the structure of robOS as seen in the presentation. So this should be improved in a future release of robOS.